src/item.h

```diff
@@ -696,6 +706,9 @@ class item : public visitable<item>
         /** remove frozen tag and if it takes freezerburn, applies mushy/rotten */
         void apply_freezerburn();
 
+        phase_id get_current_phase() const {
+            return static_cast<phase_id>( current_phase );
```

Comment:
Why is the type of `current_phase` not directly `phase_id`? That would remove the need to cast it here.

Reply:
Pretty much because I was lazy and couldn't figure out the json archive io stuff to get it to read a phase_id like an int.  I'll dig through it some more because I hate this too.